.github/workflows/build.yml

```diff
@@ -0,0 +1,58 @@
+name: build
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    permissions: # required by aws-actions/configure-aws-credentials
+      id-token: write
+      contents: read
+      checks: write
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - uses: actions/setup-java@v3
+      with:
+        distribution: 'corretto'
+        java-version: '8'
```

Comment:
Should be `11` no?

Reply:
I've had some issues converting pipelines from teamcity to GHA, so in the first instance, I normally copy whatever I find from the teamcity environment variables, so in this case, I copied this from [here](https://teamcity.gutools.co.uk/buildConfiguration/Mobile_MobileNotificationsContent2?buildTypeTab=buildTypeSettings#all-projects):
<img width="1522" alt="image" src="https://github.com/guardian/mobile-notifications-content/assets/14816091/7463ab8e-8bca-4a29-ab1d-5cebd13f2a2c">

However I did move to corretto, someone gave the guidance that this didn't matter a lot, but that corretto was more closely aligned with AWS, which is where this would be deployed.

Reply:
At the moment it looks like we specify java 8 in the [build.sbt](https://github.com/guardian/mobile-notifications-content/blob/bac9ba03561d3b0bd5a3118a9ef6ad4df3e7e99d/build.sbt#L14), so if it's ok we can migrate/test the use of java 11 as a separate task? I've created a [ticket](https://theguardian.atlassian.net/browse/LIVE-5433) for our health backlog so we can prioritise this.

Reply:
@DavidLawes that make sense, thanks!